src/App.tsx

```diff
@@ -1,13 +1,17 @@
-import { HStack } from "@chakra-ui/react";
+import { HStack, VStack } from "@chakra-ui/react";
 import MapContainer from "./common/map/containers/MapContainer";
 import SidebarContainer from "./common/sidebar/containers/SidebarContainer";
+import Header from "./common/components/header/Header"
 
 function App() {
   return (
-    <HStack bg="gray" height="100vh">
-      <SidebarContainer width="35%" />
-      <MapContainer />
-    </HStack>
+    <VStack align="stretch" bg="orange" >
```

Comment:
We can probably remove the orange and I only had that so it was easier to visualize, so they should be removed as the components were completed